cip/CIP2015-10-12-CREATE.adoc

```diff
@@ -0,0 +1,136 @@
+= CIP2015-10-12 - CREATE
+:numbered:
+:toc:
+:toc-placement: macro
+:source-highlighter: codemirror
+
+*Authors:* Pontus Melke pontus.melke@neotechnology.com, Mats Rydberg mats@neotechnology.com
+
+[abstract]
+.Abstract
+--
+This proposal formalizes `CREATE` which is used in Cypher to create nodes and relationships.
```

Comment:
I'll mention that these two things, `CREATE UNIQUE` and `CREATE CONSTRAINT` are _not_ included.